atox/src/main/kotlin/App.kt

```diff
@@ -16,4 +19,23 @@ class App : MultiDexApplication() {
 
     @VisibleForTesting
     var componentOverride: AppComponent? = null
+
+    @Inject
+    lateinit var callManager: CallManager
+
+    @Inject
+    lateinit var proximityScreenOff: ProximityScreenOff
```

Comment:
I think this deployment should be moved to ToxService. No need to run this at app startup; running it when the Tox service is created should be enough.